release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 7. | **Feature** | Support for parallel recursive queries to Service Fabric DNS Service | **Brief Description**: This change adds support for parallel recursive queries in SF DNS, where requests will be sent to all forwarding DNS servers and the first response received is used. When enabled, parallel queries will be attempted after the serial queries. The following config options are introduced under **DNS Service** to control the behavior:<br><ul><li>**RecursiveQuerySerialMaxAttempts** - The number of serial queries that will be attempted, at most. If this number is higher than the amount of forwarding DNS servers, querying will stop once all the servers have been attempted exactly once.</li><li>**RecursiveQuerySerialTimeout** - The timeout value in seconds for each attempted serial query.</li><li>**RecursiveQueryParallelMaxAttempts** - The number of times parallel queries will be attempted. Parallel queries are executed after the max attempts for serial queries have been exhausted.</li><li>**RecursiveQueryParallelTimeout** - The timeout value in seconds for each attempted parallel query.</li></ul>If clusters were previously using the RecursiveQueryTimeout config option, they will need to remove it before upgrading; this config is replaced by the ones above.<br>**Documentation Reference**: Read more about it at - <ul><li>[DNS Service in Azure Service Fabric - Recursive Queries](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-dnsservice#recursive-queries)</li></ul> |
+| 8. | **Feature** | Support for Managed KeyVaultReference | **Brief Description**: Service Fabric's auto-rotation story for application secrets is here: Managed KeyVaultReferences. Leverage Managed Identities for SF Apps, and KeyVaultReferences to get automatic application parameter upgrades as your secrets rotate in the vault.<br>**Documentation Reference**: Read more about it at - <ul><li>[Use KeyVaultReferences in your application](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-keyvault-references#use-keyvaultreferences-in-your-application), and </li><li>[Use Managed KeyVaultReferences in your application](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-keyvault-references#use-managed-keyvaultreferences-in-your-application)</li></ul> |
```

Comment:
Service Fabric's auto-rotation story for application secrets is here: Managed KeyVaultReferences. -> 

Managed KeyVaultReference is Service Fabric's auto-rotation application secrets.